### DISCUSSION TITLE
QR Timecode sync between Hero11 and Hero12 is consistently off by ~1/4 second

### DISCUSSION THREAD

**Discussion by RobertMarkEnger** [gopro/labs#830] [General]
I wanted to use a Hero11 and Hero12 as a 2-camera set-up with frame-level synchronization between them.

I loaded LABS firmware on both.  I oriented them so that BOTH cameras could see the QR code simultaneously.

I repeated the following test three times:

- Power-up both cameras; 
- present QR and watch cameras acknowledge setting TC
- quasi-simultaneously push record on both cameras 
- shoot a minute or so of video with a loud time-marker event (me clanging two metal rods together) 
- stop cameras 
- wait briefly (for files to write-out, etc)
- power-off both cameras.  

When I had done that three times, I loaded the files into Davinci Resolve Pro using align-by-timecode, and examined the sound traces.
The two cameras are consistently off by ~6 frames at "24" fps. 
 
GoPro has recently suggested that groups of GoPro cameras can be synchronized for multi-camera shoots. 
This testing undercuts their assertion.  (Maybe it works better if both cameras are of the same model?)

Attached are screen-caps showing the offsets.
![Offset_trial-1](https://github.com/gopro/labs/assets/157267205/e4400f56-1f44-410f-ac3b-cfe015c48c0d)
![Offset_trial-2](https://github.com/gopro/labs/assets/157267205/2cf60350-b2fe-4cce-9de6-87671d84de23)
![Offset_trial-3](https://github.com/gopro/labs/assets/157267205/87795cb8-d6dd-4b0a-b162-d0b513c54a08)

**Comment by dnewman-gpsw** [comment]
This is to be expected between camera models, as the firmware changes the response time changes.  Fortunately there is a workaround with TCAL  - Timecode Calibration.  Documentation is here: https://gopro.github.io/labs/control/extensions/
